packages/dev/buildTools/src/prepareEs6Build.ts

```diff
@@ -12,7 +12,7 @@ export const prepareES6Build = async () => {
             const constantsContent = fs.readFileSync(path.resolve(baseDir, constFile as string), "utf8").replace("export class Constants", "const Constants = ");
             // eslint-disable-next-line @typescript-eslint/naming-convention
             const Constants = eval(constantsContent + "\nConstants;");
-            const allSourceFiles = glob.sync(path.resolve(baseDir, "**", "*.js"));
+            const allSourceFiles = globSync(path.resolve(baseDir, "**", "*.js"));

```

Comment:
Does this one need the flag also?

Reply:
yep! missed this one for some reason. thanks!

Reply:
Done :-)